src/components/FullPlayground.tsx

```diff
@@ -756,59 +727,59 @@ export function ThemedAppView(props: { datastore: DataStore }) {
               )}
             </div>
             {AppConfig().discord.inviteUrl ? (
-              <Button
-                className={classes.hideTextOnMed}
-                size="small"
-                href={AppConfig().discord.inviteUrl}
-                startIcon={
+              <Button asChild variant="link" size="sm">
+                <a
+                  href={AppConfig().discord.inviteUrl}
+                  target="_blank"
+                  rel="noreferrer"
```

Comment:
`noreferrer` isn't necessary but is generally recommended when opening new tabs. It prevents the target tab from gaining context about the parent.